z3rro/src/prover.rs

```diff
@@ -21,6 +25,68 @@ pub enum ProveResult<'ctx> {
     Unknown(ReasonUnknown),
 }
 
+/// Find the swine-z3 file located under the dir directory, and execute swine-z3 on the file located at file_path
+fn execute_swine(dir: &Path, file_path: &Path) {
+    let swine = "swine-z3";
+
+    let find_output = Command::new("find")
```

Comment:
I don't understand why you'd be using `find` here? It should be sufficient to just do the call to swine directly.